device-types/F5/BIG-IP-i7800.yml

```diff
@@ -0,0 +1,52 @@
+---
+manufacturer: F5
+model: BIG-IP i7800
+slug: f5-big-ip-i7800
+part_number: i7800
+u_height: 1
+is_full_depth: true
+weight: 30
+weight_unit: lb
+airflow: front-to-rear
+front_image: true
```

Comment:
if you set `front_image: true` you need to add the corresponding `<slug>.jpg` in `eleveation-images/F5/`

Reply:
I will add the images and update the pull request when I have a minute.  I might not get to it before monday, though.

Reply:
Images have been added.  I had to use some physical images as the ones on the f5 site are all taken from one corner making it look skewed.